ui/imports/shared/views/PickedContacts.qml

```diff
@@ -65,6 +62,8 @@ Item {
             statusListItemIcon.badge.border.color: Theme.palette.baseColor4
             statusListItemIcon.badge.implicitHeight: 14 // 10 px + 2 px * 2 borders
             statusListItemIcon.badge.implicitWidth: 14 // 10 px + 2 px * 2 borders
+
+            MouseArea { anchors.fill: parent }
```

Comment:
Why is this here?

Reply:
Let me not answer that question. Already fixed ;)